src/pages/404.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Heading, Stack, VStack, Text } from '@chakra-ui/react'
+import { NextSeo } from 'next-seo'
+import Image from 'next/image'
+import NextLink from 'next/link'
+
+import mascotLogo from '@app-public/mascot-logo.png'
+import { CTAButton } from '~/components/CTAButton'
+import FluidContainer from '~/components/FluidContainer'
+import Layout from '~/components/Layout'
+import Link from '~/components/Link'
+
+const Custom404 = () => (
+  <Layout>
+    <NextSeo title="404 - Not Found" />
+    <Box pb={{ base: 8, lg: 16 }} align="center">
+      <FluidContainer>
+        <VStack px="10">
+          <Image
+            src={mascotLogo}
+            alt="Octoclairvoyant reading a crystal ball"
+            placeholder="blur"
+            priority
+            width={250}
+            height={250}
+          />
+          <Stack
+            shouldWrapChildren
+            alignItems="center"
+            spacing="2"
+            flexGrow={1}
+            flexBasis={0}
+          >
+            <Heading
+              as="h2"
+              color="gray.900"
+              fontSize="4xl"
+              fontWeight="black"
+              letterSpacing="tight"
+            >
+              This page could not be found.
+            </Heading>
+            <Text as="p" color="coolGray.700" fontSize="2xl" fontWeight="black">
+              Octoclairvoyant cannot divine the page you wanted.
+            </Text>
+          </Stack>
+        </VStack>
+        <VStack mt={12} spacing={5}>
+          <NextLink href="/comparator">
+            <CTAButton>Go to comparator</CTAButton>
```

Comment:
```suggestion
            <CTAButton>Go to repository search</CTAButton>
```

I think "Go to repository search" is a better description than "Go to comparator". If you hit the 404 page, something went wrong, and we want people to recover quickly themselves.

Maybe using "Search" in the button is clearer on the action they can take after a user clicks the button?

If we put the `Go to repository search` into the button, it does break down on iPhone SE (1st gen) or smaller devices, where the button becomes too big too fit on screen comfortably.

I'm not sure how many viewports we want to define, each one we define is maintenance work.... So maybe we can just go with your text after all, and rely on `less text = smaller button`... 😄

Reply:
Not really sure. I just realized your designs had the "Go to repository search" text, but it was natural for me to put "Go to comparator". I think we should refer to the page itself, which is the comparator. I find "repository search" quite confusing: it could be the comparator page starting by searching a repository, or another page to search repositories? I'd stick to "comparator", to be honest.

Reply:
The idea was that people who land on the 404 might not even know what a comparator is, so that's why I thought maybe we could use `Go to repository search`, as that might be clearer for people new to Octoclairoyant.

The button is nice and big, so it'll be the first thing people will hit anyways, so maybe the text is less important.

It's also important to use a single word for a single concept, so let's keep `comparator` then. 😉